src/LINEBot/Event/BaseEvent.php

```diff
@@ -194,4 +194,14 @@ public function getEventSourceId()
         # Unknown event
         return null;
     }
+
+    /**
+     * Returns the instance as an array.
+     *
+     * @return array
+     */
+    public function toArray()
```

Comment:
The toArray name may be looks like cast to array data recursively, but there are no `arrayable` string data like json in webhook events.

Reply:
How about naming `getEvent()` instead?

Reply:
I considered that $event->getEvent() may confuse that meanings because $event property is private.
If it's unfounded, I think `getEvent()` naming is better.

Reply:
Exposing this variable might be helpful for debugging purposes.
Even if it's private variable, user still can not change it's value.

Reply:
I think so too. I renamed the method `getEvent()`.